### ISSUE TITLE
Tab by target and then forecast date in reports

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/covid19-forecast-hub-europe#564]
At the moment the report is tabbed by forecast date and then target. This can be quite annoying if you are comparing death forecasts across time. A simple solution would be to reverse the tab order but this may not be ideal for people wanting to look at cases and deaths forecasts for a certain date. That isn't how I use the reports but maybe it is how some people do. 

Another option is a different organization of the forecasts entirely but I aam not clear what that is.

**Comment by Bisaloo** [comment]
I agree that the double tab navigation is a bit awkward in general (hence the related issue #563). I think it would be worth trying to think about a different organization entirely, as you mention. I will have a look of some options.
